tox.ini

```
@@ -53,7 +53,7 @@ commands_pre =
   pytest tests/test_sparql.py
 
 commands =
-  python model/model.py
+  python model/model.py --sparql http://localhost:18890/sparql
```

```suggestion
  python model/model.py
```

Tests should be run inside docker, in a consistent environment